docs/topics/spiders.rst

```diff
@@ -145,6 +145,26 @@ scrapy.Spider
        :param kwargs: keyword arguments passed to the :meth:`__init__` method
        :type kwargs: dict
 
+   .. method:: update_settings()
+
+       The ``update_settings`` method allows you to modify the settings of a spider. It is a classmethod that is called to 
+       override the configuration of your spider. When called, it receives the spider's settings and passes ``custom_settings``
+       or an empty dictionary to the ``setdict`` method to register the setting with a priority of spider. It is recommended to
```

Comment:
Everything starting with “and passes” to the end of the sentence feels like an implementation detail, not what a user needs to know here. The user needs to know that they can edit the settings on the received `settings` object, and at what point in time this method is called in relation to other settting-defining code (e.g. are the update_settings classmethods of Scrapy components like middlewares called before or after this class method?).